robot_name: comau                     # iiwa               panda
arm_name: comau_arm                   # iiwa_arm           panda_arm
left_arm_name: left_comau_arm         # left_iiwa_arm      left_panda_arm
right_arm_name: right_comau_arm       # right_iiwa_arm     right_panda_arm
both_arm_name: both_comau_arm         # both_iiwa_arm      both_panda_arm
left_end_effector: comau_left_link_ee   # left_link_ee       panda_left_link8
right_end_effector: comau_right_link_ee # right_link_ee      panda_right_link8
left_joint: comau_left_joint_          # left_joint_        panda_left_joint
right_joint: comau_right_joint_        # right_joint_       panda_right_joint
arm_name_left_temp: comau_left        # left               panda_left
arm_name_right_temp: comau_right      # right              panda_right
joint_name_temp: _joint_               # _joint_            _joint
num_joints: 6